dom.bs

```diff
@@ -1419,6 +1419,142 @@ that gave folks all the wrong ideas. <a>Events</a> do not represent or cause act
+providing an {{AbortController/abort()}} method that toggles the state of a corresponding
+{{AbortSignal}} object. The API which wishes to support aborting can accept such a {{AbortSignal}},
+and use its state to determine how (not) to proceed.
+
+APIs that rely upon {{AbortController}} are encouraged to respond to {{AbortController/abort()}} by
+rejecting any unsettled {{Promise}} with a new {{DOMException}} with [=error name=]
+"{{AbortError}}".
+
+<div class=example id=aborting-ongoing-activities-example>
+ For example, a hypothetical <code>doAmazingness({ ... })</code> method could accept a
+ {{AbortSignal}} object in order to support aborting as follows:
+
+ <pre class=lang-javascript>
+  const controller = new AbortController();
+  const signal = controller.signal;
+
+  startSpinner();
+
+  doAmazingness({ ..., signal })
+    .then(result => ...)
+    .catch(err => {
+      if (err.name == 'AbortError') return;
+      showUserErrorMessage();
+    })
+    .then(() => stopSpinner());
+
+  ...
+
+  controller.abort();
+ </pre>
+
+ <code>doAmazingness</code> could be implemented as follows:
+
+ <pre class=lang-javascript>
+  function doAmazingness({signal}) {
+    return new Promise((resolve, reject) => {
+      // Begin doing amazingness, and call resolve(result) when done.
+      // But also, watch for signals:
+      signal.addEventListener('abort', () => {
+        // Stop doing amazingness, and:
+        reject(new DOMException('Aborted', 'AbortError'));
+      });
+    });
+  }
+ </pre>
+
+ APIs that require more granular control could extend both {{AbortController}} and {{AbortSignal}}
+ according to their needs.
+</div>
+
+<h3 id=interface-abortcontroller>Interface {{AbortController}}</h3>
+
+<pre class="idl">
+[Constructor(), Exposed=(Window,Worker)]
+interface AbortController {
+  [SameObject] readonly attribute AbortSignal signal;
+
+  void abort();
+};
+</pre>
+<dl class=domintro>
+ <dt><code><var>controller</var> = new <a constructor lt="AbortController()">AbortController</a>()</code>
+ <dd>Returns a new <var>controller</var> whose {{AbortController/signal}} is set to a newly
+ created {{AbortSignal}} object.
+
+ <dt><code><var>controller</var> . </code>{{AbortController/signal}}
+ <dd>Returns the {{AbortSignal}} object associated with this object.
+
+ <dt><code><var>controller</var> . </code>{{AbortController/abort()}}
+ <dd>Invoking this method will set this object's {{AbortSignal}}'s [=AbortSignal/aborted flag=],
+ thereby signaling to any observers that the associated activity should be aborted.
+ </dl>
+
+The <dfn attribute for=AbortController><code>signal</code></dfn> attribute must return the value to
+which it was initialized. When a {{AbortController}} is created, the attribute must be initialized
+to a newly created {{AbortSignal}} object.
+
+The <dfn method for=AbortController><code>abort()</code></dfn> method, when invoked, must <a
+for=AbortSignal>signal abort</a> on this object's {{AbortController/signal}}.
```

Comment:
You can probably write the link as `[=AbortSignal/signal abort=]`, although I haven't tested that with DOM's build system.